public/apps/account/account-app.tsx

```diff
@@ -36,6 +37,20 @@ function tenantSpecifiedInUrl() {
 }
 
 export async function setupTopNavButton(coreStart: CoreStart, config: ClientConfigType) {
+  const authType = config.auth?.type;
+  let currAuthType = '';
```

Comment:
Should the `currAuthType ` be initialized with the default authType(`AuthType.BASIC`) instead of an empty string?

Reply:
I would recommend to use empty string. Exception will only happen when the api call fail due to internal errors. In order to handle this exception, we allow users to log out locally. In theory, currentAuthType should always be set by the api. We should not set a default value for the currentAuthType.

Reply:
After discussing with @aoguan1990 offline, `''`, `'basicauth' and `AuthType.BASIC` all end up with pointing to basi c auth. Thare are quite a few occurrences in the code base. I'm creating an issue for clean-up out of this PR.

Reply:
created an issue https://github.com/opensearch-project/security-dashboards-plugin/issues/1160

Reply:
@aoguan1990 Would you please create tracking issues for the other items that need follow-up changes? I recall that we have a few more in the comments that have already been marked as resolved.

Reply:
Sure.